io.catenax.early_warning_notification/1.0.0/EarlyWarningNotification.ttl

```diff
@@ -0,0 +1,172 @@
+   samm:description "List of unique identifiers to explicitly specify parts or vehicles exhibiting the issue described in the property \"Information\""@en ;
+   samm:characteristic :AffectedItemsCharacteristic .
+
+:poulationFilterList a samm:Property ;
+   samm:preferredName "Population Filter List"@en ;
+   samm:description "A set of Population Filters"@en ;
+   samm:characteristic :PopulationFilterListCharacteristic .
+
+:earlyWarningAttachmentLink a samm:Property ;
+   samm:preferredName "Additional Quality Data Link"@en ;
+   samm:description "This property provides a reference to an EDC asset that will contain a zip file organized according to the specification CX-0092 and the model defined in io.catenax.quality_task_attachment:1.0.0. The zip file will contain additional data describing the early warning case."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "subprotocolBody: catalogEndpoint=https://test.com/catalog;dataset=79afc338-f7ea-4255-a17d-ba6faf40d2b5" .
+
+:Status a samm-c:Enumeration ;
+   samm:preferredName "Status  "@en ;
+   samm:description "A status infomration d´selected from a predefined set of possible states."@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "ACKNOWLEDGED" "ACCEPTED" "DECLINED" "CLOSED" ) .
+
+:Severity a samm-c:Enumeration ;
+   samm:preferredName "Severity"@en ;
+   samm:description "Describes the severity of a quality issue."@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "MINOR" "MAJOR" "CRITICAL" "LIFE-THREATENiNG" ) .
+
+:AffectedItemsCharacteristic a samm-c:List ;
+   samm:preferredName "Affected Items Characteristic"@en ;
+   samm:description "Characteristic for the UIDs defining the affected items"@en ;
+   samm:dataType :Items .
+
+:PopulationFilterListCharacteristic a samm-c:List ;
+   samm:preferredName "Population Filter List Characteristic"@en ;
+   samm:description "Characteristic for a List of Population Filters"@en ;
+   samm:dataType :PopulationFilter .
+
+:Items a samm:Entity ;
+   samm:preferredName "Items"@en ;
+   samm:description "A unique identifier"@en ;
+   samm:properties ( [ samm:property :anonymizedVIN; samm:optional true ] [ samm:property :catenaXId; samm:optional true ] [ samm:property :serialPartNumber; samm:optional true ] ) .
+
+:PopulationFilter a samm:Entity ;
+   samm:preferredName "Population Filter"@en ;
+   samm:description "Population filter for the early warning data"@en ;
+   samm:properties ( :aspectProperty :aspectModel [ samm:property :rangeFrom; samm:optional true ] [ samm:property :rangeTo; samm:optional true ] :dataType [ samm:property :values; samm:optional true ] ) .
+
+:anonymizedVIN a samm:Property ;
+   samm:preferredName "VAN"@en ;
+   samm:description "A unique but anonymized identifier for a vehicle"@en ;
+   samm:see <urn:samm:io.catenax.vehicle.product_description:3.0.0> ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "3747429FGH382923974682" .
+
+:catenaXId a samm:Property ;

```

Comment:
if this is a unique catenaXId,  you can make use of shared aspect model Uuid.
example: https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/io.catenax.asset_tracker_links/2.0.0/AssetTrackerLinks.ttl#L38

Reply:
okay, i misunderstood, I changed it